04_verify.sh

```diff
@@ -253,7 +263,14 @@ RESULT_STR="Fetch CRDs"
 CRDS="$(kubectl --kubeconfig "${KUBECONFIG}" get crds)"
 process_status $? "Fetch CRDs"
 
-if [[ "${CAPI_VERSION}" == "v1alpha2" ]] || [[ "${CAPI_VERSION}" == "v1alpha3" ]]; then
+if [ "${CAPI_VERSION}" == "v1alpha3" ]; then
```

Comment:
Similar comment as above. May be we should populate the list of CRDs based on the version once and then run the check once only rather than having similar code for all the versions.

Reply:
done